### ISSUE TITLE
Animation hook names as props to avoid global css

### ISSUE THREAD

**Issue by ankitduseja** [jl-/react-slide-deck#10]
I am using CSS Modules and the animation hooks are currently static global classes. I think it will be good to have hook names be customised by passing them as props to the `<Deck>` and would help in avoiding global css completely.

**Comment by jl-** [comment]
Hi @ankitduseja , thanks for bringing it up, and that's what should've been done  at the beginning. I just made a new push to the demo, take a look at [it](http://jl-.github.io/react-slide-deck/), and [readme](https://github.com/jl-/react-slide-deck/tree/gh-pages#usage) here

**Comment by ankitduseja** [comment]
Passing them individually to all children `<Slide>` makes it cumbersome, can we probably pass it once to the `<Deck>` to avoid repetition. Also just to avoid possible typos/confusion between className and classNames, a better prop name would like deckClasses/deckProps or slideClasses/slideProps.

**Comment by jl-** [comment]
Right, `classNames` is really bad and confused. And on where to put this prop, I just imprudently go by the quick demo, should've been more careful. 
I prefer `<Deck slideClasses>`,
